index.js

```diff
@@ -18,7 +18,7 @@ app.use(cors());
 
 app.use(
   session({
-    secret: "keyboard cat", //<---- put this in config, its in a public repo rn
+    secret: process.env.SESSION_SECRET,
```

Comment:
Nice, might as well add it to env.example too so when you clone this repo its simple to get up and running...